repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -388,12 +517,15 @@ class Interpreter(prompt0: Ref[String],
     @tailrec def loop(blocks: Seq[(String, Seq[String])],
                       scriptImports: Imports,
                       lastImports: Imports,
-                      wrapperIndex: Int): Res[Imports] = {
+                      wrapperIndex: Int,
+                      compiledData: List[CacheDetails]
+                     ): Res[(Imports, List[CacheDetails])] = {
```

Comment:
`(Imports, List[CacheDetails])` should be its own case class, since we appear to be using it everywhere. Put it in `Util.scala` with all the other such case classes